scripts/publish.sh

```diff
@@ -309,6 +309,11 @@ sync_latest_image() {
 		create_manifest_list_init ${account_id}.dkr.ecr.${region}.${endpoint}/aws-for-fluent-bit "init-latest" ${AWS_FOR_FLUENT_BIT_VERSION_DOCKERHUB}
 		create_manifest_list_init ${account_id}.dkr.ecr.${region}.${endpoint}/aws-for-fluent-bit "$init"-${AWS_FOR_FLUENT_BIT_VERSION_DOCKERHUB} ${AWS_FOR_FLUENT_BIT_VERSION_DOCKERHUB}
 	fi
+
+	if [ "${AWS_FOR_FLUENT_BIT_STABLE_VERSION}" != "${AWS_FOR_FLUENT_BIT_VERSION_DOCKERHUB}" ]; then
+		create_manifest_list ${account_id}.dkr.ecr.${region}.${endpoint}/aws-for-fluent-bit ${AWS_FOR_FLUENT_BIT_STABLE_VERSION} ${AWS_FOR_FLUENT_BIT_STABLE_VERSION}
```

Comment:
just checking, creating the manifest for the stable version tag over and over again is safe right? I think it is..

Reply:
If you were to create the manifest locally for the same tag twice you would be alerted to use a the `--amend` flag. Since this is handled via a serverless solution we should not have leftover manifest locally. After the local manifest is created the push command will update what is on the repo and I have yet to get that to complain about a duplicate.